bm-persona/Map/MapMarkerDetailView.swift

```diff
@@ -221,6 +226,16 @@ enum MapMarkerDetail {
             label.textColor = Color.primaryText
             label.text = description
             return viewWithIcon(icon, view: label)
+        case .price:
+            guard let price = marker.mealPrice else { return nil }
+            let icon = UIImage(named: "Dining")
```

Comment:
the icon looks kinda "squashed"/pixelated/cut off on my screen. the dining icon when you search a dining hall is pretty high-res, so I think this should be a solvable issue?
the clock and pin icons on the detail view also look a bit pixelated

Reply:
![IMG_1723](https://user-images.githubusercontent.com/20230668/93035018-b96b6200-f5f0-11ea-812e-670ec7c8ea8b.PNG)

Reply:
Fixed!